buildscripts/dbtests.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+db="$1"
+
+case ${db} in
+  mysql*)
+    db="mysql"
+    dbContainerOpts="--name mysql_tests mysql mysqld --innodb_file_per_table"
+    DBURL="server@tcp(mysql_tests:3306)/notaryserver?parseTime=True"
+    ;;
+  rethink*)
+    db="rethink"
+    dbContainerOpts="--name rethinkdb_tests rdb-01 --bind all --driver-tls-key /tls/key.pem --driver-tls-cert /tls/cert.pem"
+    DBURL="rethinkdb_tests"
+    ;;
```

Comment:
non-blocking nit: would be nice to have a default error case, something like:

```
       *)
            echo $"Usage: $0 dbName"
            exit 1
```

Currently I get the following if I don't provide any arg
```
...
+ docker-compose -p dbtests_ -f development..yml kill
ERROR: .IOError: [Errno 2] No such file or directory: u'./development..yml'
```